### PR TITLE
chore(*) improve resource manager initialization readability

### PR DESCRIPTION
**Pull request by jpeach**
### Summary

This refactoring is a bit gratuitous, so I'm not that concerned if we don't want to merge it :-)

I was reading the resource manager setup and it was confusing how the type map was still being initialized after already being "consumed" by customizable resource manager.

### Full changelog

N/A

### Issues resolved

N/A

### Documentation

N/A

### Testing

- [x] Unit tests
- [x] E2E tests
- [ ] Manual testing on Universal
- [ ] Manual testing on Kubernetes

### CODE REVIEW

File: pkg/core/resources/manager/customizable_manager.go [hunk 1]
```diff
@@ -28,6 +32,9 @@ type customizableResourceManager struct {
 }
 
 func (m *customizableResourceManager) Customize(resourceType model.ResourceType, manager ResourceManager) {
+	if _, ok := m.customManagers[resourceType]; ok {
+		panic(fmt.Sprintf("resource type %q is already customized", resourceType))
```
Comment (lobkovilya) [inline pkg/core/resources/manager/customizable_manager.go:36]: I'd better panicked with `error` rather than `string`

**Review by lobkovilya — APPROVED** [review]
This makes sense and for sure improves readability, so I'm not against merging this one

**Review by jakubdyszkiewicz — CHANGES_REQUESTED** [review]
My bad, I introduced Customize method, I should have refactored the rest.

👍

File: pkg/core/resources/manager/customizable_manager.go (hunk 1 above)
Comment (jpeach) [reply pkg/core/resources/manager/customizable_manager.go]: I did (1)

**Review by jakubdyszkiewicz — APPROVED** [review]

**Pull request merged**